test/tests/misc/max_var_n_dofs_per_elem/tests

```diff
@@ -35,15 +35,24 @@
     design = "MaxVarNDofsPerElem.md"
     issues = "#5658"
   [../]
-  [./triangles_high_order]
+  [./triangles_second_order]
     type = 'Exodiff'
     input = 'max_var_n_dofs_per_elem.i'
-    exodiff = 'triangles_high_order.e'
-    cli_args = 'Mesh/dim=2 Mesh/elem_type=tri6 Variables/u/order=SECOND Outputs/file_base=triangles_high_order'
+    exodiff = 'triangles_second_order.e'
+    cli_args = 'Mesh/dim=2 Mesh/elem_type=tri6 Variables/u/order=SECOND Outputs/file_base=triangles_second_order'
     requirement = "The maximum number of degrees of freedom for a single variable on a 2D TRI6 elem shall be 6"
     design = "MaxVarNDofsPerElem.md"
     issues = "#5658"
   [../]
+  [./triangles_third_order]
+    type = 'Exodiff'
+    input = 'max_var_n_dofs_per_elem.i'
+    exodiff = 'triangles_third_order.e'
```

Comment:
are we testing the postprocessor here?
If it should be a csvdiff not an exodiff 

this is probably because it s a really old test

Reply:
Done for the existing tests in #25727.
I'll rebase and apply the same changes here once that's merged. :)